### PR TITLE
feat: add card component

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+.card-head-box .badge {
+    padding: .2rem 1rem;
+    position: absolute;
+    font-size: 1rem;
+    color: var(--secondary-color);
+    background-color: var(--primary-color);
+    top: -1rem;
+    left: -1rem;
```
Comment (VishalPatil18) [inline components/card/card.css:69]: A suggestion here would be that you could sort all these properties in alphabetical order as that helps in debugging as well as code maintainability.

**Review by VishalPatil18 — APPROVED** [review]
Just one suggestion added. Apart from that everything looks good and works great. Good work Savita 👍

File: components/card/card.css (hunk 1 above)
Comment (vermasavita) [reply components/card/card.css:69]: Okay, noted

**Review by ruchiragoswami — APPROVED** [review]
Great work Savita!

**Pull request merged**